test/groovy/HadolintExecuteTest.groovy

```diff
@@ -0,0 +1,61 @@
+import hudson.AbortException
+
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.rules.RuleChain
+import util.*
```

Comment:
[minor] I vote for qualified imports since there are situation where it is hard to see which class is finally used.